lib/array.js

```diff
@@ -148,11 +148,11 @@ module.exports = (api) => {
         done(err);
       } else {
         count++;
-        if (count === len) done();
+        if (count === len) done(null);
       }
     };
 
-    if (len < 1) return done();
+    if (len < 1) return done(null);
```

Comment:
Because user expects `err` in callback to be `null`. I've never met function which gives `undefined` as `err` argument to a callback.
I understand that for code like that below it's no difference.
```javascript
const callback = (err, res) => {
  if (err) return err;
  return res;
}
```